frontend/src/components/NavBar.js

```diff
@@ -14,18 +14,25 @@ class navbar extends Component {
 						<li className={css(styles.titleelement)}><Link className={css(styles.links)} to="/">Hissues</Link></li>
 						<li className={css(styles.navelement)} onClick={() => this.props.logout()}><Link className={css(styles.links)} to="/logout">Logout</Link></li>
 						<li className={css(styles.navelement)}><Link className={css(styles.links)} to="/dashboard">Dashboard</Link></li>
-						<li className={css(styles.navelement)}><Link className={css(styles.links)} to="/newissue">New Issue</Link></li>
-						<li className={css(styles.navelement)}><Link className={css(styles.links)} to="/newhouse">New House</Link></li>
 					</ul>
 				</div>
 			);
+			// can't get these to work with redux. problem is inital state on new page
+			// <li className={css(styles.navelement)}><Link className={css(styles.links)} to="/newissue">New Issue</Link></li>
+			// <li className={css(styles.navelement)}>
+			// 	<Link className={css(styles.links)} to={{
+			// 			pathname: "/house",
```

Comment:
why are these comments here?

Reply:
I have commented them out in case we find a way around the problem I was having with new house/issue on the menu bar.